src/main/java/baritone/pathing/movement/MovementHelper.java

```diff
@@ -344,6 +347,25 @@ static boolean canWalkOn(BlockStateInterface bsi, int x, int y, int z, IBlockSta
         return block instanceof BlockStairs;
     }
 
+    static List<Item> filterThrowawayForPlacing(List<Item> items) {
```

Comment:
I think a comment here (and probably on `MovementHelper::canWalkOn` as well) would be a good idea to make sure people know these two methods  should be roughly in sync (precisely: this method may only return items where `MovementHelper::canWalkOn` is guaranteed to return `true` for all blockstates the item can place).